star-rx72n-firmware/lib/rx_hcsr04/src/rx_hcsr04.c

```
@@ -762,6 +841,9 @@ float rx_hcsr04_get_speed_of_sound(float temp_celsius)
 
 float rx_hcsr04_echo_to_cm_with_temp(uint32_t echo_time_us, float temp_celsius)
 {
+  float speed_mps   = 0.0f;
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)